test/make-images-push-to-local-registry.sh

```diff
@@ -12,3 +12,5 @@ docker tag cilium/operator:$2 $1/cilium/operator:$2
 docker push $1/cilium/cilium:$2
 docker push $1/cilium/cilium-dev:$2
 docker push $1/cilium/operator:$2
+
+docker image prune -f --all --filter "label=cilium-sha=$(cat GIT_VERSION | cut -d' ' -f1 | tr -d '\n')"
```

Comment:
```
cilium_git_version="$(cat GIT_VERSION)"
docker image prune -f --all --filter "label=cilium_git_version={git_version%% *}"`
```

Reply:
Apologies for the typo, I fixed it now for the record, and thanks for accepting this suggestions :)